corehq/apps/hqadmin/management/commands/check_case_index_ids.py

```diff
@@ -108,7 +108,7 @@ def handle(self, **options):
 
             for case_to_resync in exists:
                 # if the case actually exists resync it to fix the es search
-                resync_case_to_es(domain, case_to_resync)
+                resave_case(domain, case_to_resync)
```

Comment:
Looking at this closely now, this probably is a bug.
this wants to just resync case but is actually doing two things
1. resync es
2. also send post case save signal.
it may be does not need the latter. Should this be updated now to `resave_case(domain, case_to_resync, send_post_save_signal=False)`

Reply:
yes I think so